LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DeviceLoaderSynchronizer.cs

```diff
@@ -88,7 +91,11 @@ internal enum LoaderState
                     continuationAction(t, this);
                 }
             },
-            ex => this.logger.LogError($"Error while loading: {ex}."));
+            ex =>
+            {
+                this.logger.LogError($"Error while loading: {ex}.");
+                processingErrorCount?.Add(1);
```

Comment:
[nit] What if we add this in the TaskUtil.RunOnThreadPool? This way we can avoid adding this in all the callers of it.

Reply:
Valid point, I changed the signature of the method to require passing a `Counter<int>`. This will make it explicit to any caller that exceptions are counted - in case of observed executions it's still up to the caller to decide whether to not pass the counter if the task is awaited.

Reply:
Makes sense to have the counter nullable 👍